oap-server/server-starter/src/main/resources/ui-initialized-templates/activemq/activemq-broker.json

```diff
@@ -0,0 +1,355 @@
+[
+    {
+        "id": "ActiveMQ-Broker",
+        "configuration": {
+            "children": [
+                {
+                    "x": 6,
+                    "y": 0,
+                    "w": 6,
+                    "h": 9,
+                    "i": "0",
+                    "type": "Widget",
+                    "expressions": [
+                        "meter_activemq_broker_state"
```

Comment:
please use the `latest or avg`... function to aggregate values if the widget doesn't show the time series values.

Reply:
https://github.com/apache/skywalking/pull/12109/commits/67673bd1c40d106a05acd341e4414a27a5d5d637
updated